Makefile

```diff
@@ -80,17 +106,30 @@ build-functest:
 functest: build-functest
 	@echo "Running functional test suite"
 	hack/functest.sh
+gofmt:
+	gofmt -s -l `find . -path ./vendor -prune -o -type f -name '*.go' -print`
 
-# This is used by the openshift-ci prow job.
-# It just makes the ocs-binary without invoking docker to build the container image.
-ocs-operator-openshift-ci-build:
-	@echo "Building ocs-operator binary for openshift-ci job"
-	mkdir -p build/_output/bin
-	go build -mod=vendor -o build/_output/bin/ocs-operator cmd/manager/main.go
+golint:
+	hack/lint.sh
+
+govet:
+	go vet ./...
+
+# ignoring the functest dir since it requires an active cluster
+# use 'make functest' to run just the functional tests
+unit-test:
+	@echo "Executing unit tests"
+	go test -v `go list ./... | grep -v "functest"`
 
 update-generated: operator-sdk
 	@echo Updating generated files
 	@echo
 	@$(OPERATOR_SDK) generate k8s
 	@echo
 	@$(OPERATOR_SDK) generate openapi
+
+verify-generated: update-generated
```

Comment:
Do you want to run the actual update here, or do you want to implement a checksum system like we did for the CSV?

Reply:
I think doing the real update should be fine here. 

The primary reason I went with the checksum approach for the csv was because I couldn't execute ```docker  run..``` in prow.  This ```make verify-generated``` target doesn't have that limitation.